app/routes/events/components/EventDetail/index.tsx

```diff
@@ -297,15 +298,15 @@ export default class EventDetail extends Component<Props, State> {
           }
         : null,
     ];
+
+    const groupLink = resolveGroupLink(event.responsibleGroup);
```

Comment:
```suggestion
    const groupLink = event.responsibleGroup && resolveGroupLink(event.responsibleGroup);
```
Seems like `event.responsibleGroup` is sometimes undefined while loading. This causes a crash, and is why the cypress tests fail.

Reply:
Ye, gotta love our SSR 🙃

Reply:
It's not really ssr-related, I think it's just caused by the way we load data with redux. I could reproduce the error without ssr enabled by just opening some different events.

Reply:
Ye I kinda realized after commenting, but I like to blame our SSR for all of my problems :)